src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSourceContext.java

```diff
@@ -147,7 +158,17 @@ public final boolean wantForkPRs() {
     public final Set<ChangeRequestCheckoutStrategy> forkPRStrategies() {
         return forkPRStrategies;
     }
-
+    /**
+     * Returns the strategies used to notify Github of build status.
```

Comment:
Document that the returned list is intended for use from the thread that owns the Context object. The list should be defensively copied if the context and the list will be managed by different threads